### DISCUSSION TITLE
Gives so many error when running dicom server locally

### DISCUSSION THREAD

**Discussion by amritanand12** [microsoft/dicom-server#1916] [Q&A]
I tried to run the dicom-server by **microsoft** locally using docker but it gives a lot of error. I am specifying the errors that i get please suggest some help for running it on my windows PC.

![BuildError-DicomServer](https://user-images.githubusercontent.com/61642500/188557696-1b5a2224-0a83-43c5-9a42-d2eca43fe0c6.jpg)

**Comment by wsugarman** [comment]
Do you have the appropriate .NET SDK installed as per the [setup instructions](https://github.com/microsoft/dicom-server/blob/main/docs/development/setup.md)? I noticed in these instructions we're missing Docker Desktop, so if you'd like to build the docker-compose project, you must have that installed as well. Otherwise, you can unload the docker-compose project to skip its build.
